include/SFML/Graphics/Font.hpp

```diff
@@ -275,6 +275,32 @@ class SFML_GRAPHICS_API Font
     ////////////////////////////////////////////////////////////
     const Texture& getTexture(unsigned int characterSize) const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Enable or disable the smooth filter
+    ///
+    /// When the filter is activated, the font appears smoother
+    /// so that pixels are less noticeable. However if you want
+    /// the font to look exactly the same as its source file,
+    /// you should disable it.
+    /// The smooth filter is enabled by default.
+    ///
+    /// \param smooth True to enable smoothing, false to disable it
+    ///
+    /// \see isSmooth
+    ///
+    ////////////////////////////////////////////////////////////
+    void setSmooth( bool smooth );
```

Comment:
Wrong spacing 😉

Reply:
:( think It must be picking up a rogue clang-format file from somewhere as this is the style I use for other projects, will fix this though ( and out of curiosity would a clang-format PR be worthwhile? )

Reply:
I would very much appreciate such a PR. Not sure how easy it is to reproduce SFML's coding style, but I'd also be willing to change some of it in favor of an automated formatting.

Reply:
Done